typescript/introduction/tsc-data-types/tsc-intro-data-types.md

```diff
@@ -0,0 +1,62 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Data Types
+
+---
+## Content
+
+A data type is defined by the values it can take and the operations that can be performed on it and . For example, `number` is a data type.
```

Comment:
````suggestion
A data type is defined by the values it can take and the operations that can be performed on it. For example, a `number` is a data type that can be subtracted but a `string` is a sequence of characters and it cannot be subtracted.

```ts
let num = 5;
let ok = num - 3; // no error

let str = 'enki';
let nono = str - 3; 
//         ^^^ 
// error because we're subtracting a string
```
````